src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -49,12 +54,32 @@ internal class IssueCreateDialog(
     }
 
     fun fillRequiredFields(): IssueCreateDialog {
+
+        driver.wait(elementToBeClickable(configColumnField)).click()
+        val locator = By.xpath("//div[@class='qf-picker-header']/dl/dd/a[text()='All']")
```

Comment:
The selector is way too specific. Do we really care about `div`s, `dl`s and `dd`s?
These page objects are supposed to work with multiple Jira versions. Let's pick the most semantic selector, which won't break after some DOM changes. Ie. let's look for the element with `Configure Fields` text and then the `All` text within it.

Reply:
in the dom tree, the "All" element is not contained within "Configure Fields" element.
Maybe I can simplify it as:
//*[@id='inline-dialog-field_picker_popup']//a[text()='All']

Reply:
Looks good 👍